.github/actions/spelling/expect.txt

```diff
@@ -1,3 +1,4 @@
+Accessbility
```

Comment:
Why is this misspelling necessary?

Reply:
It's in the code, in the obsolete function's name, so it's expected.

Reply:
I see. Can't we do such exclusions in the code itself, like with analyzers, so it's less generic (since this now enables a misspelling for all of our projects)? Or at least add this config just in the UI Testing Toolbox.

Reply:
The problem is that the spell checker breaks the name of the function into pieces, so `Accessbility` is it's own word in this context. 

I see two solutions for this problem both with upsides and downsides.

1. We ignore the UI Testing Toolbox in OSOCE and add the spelling action there on it's own.
2. We ignore the file with this method name.

What do you think?

Reply:
Both impossible.

The adding it to just the UI Testing Toolbox is possible with solution 1.

Reply:
Then it's better to keep this misspelling for now, but add a comment on the obsolete method that when we remove that, we should also remove this.